src/userservice/userservice.py

```diff
@@ -25,8 +25,13 @@
 
 import bcrypt
 import jwt
+from easy_profile import EasyProfileMiddleware
+from easy_profile.reporters import Reporter
 from flask import Flask, jsonify, request
 import bleach
+from opencensus.ext.stackdriver import stats_exporter
```

Comment:
Because opentelemetry's cloud monitoring support is incomplete, I tried using OpenCensus to export the sqlachemy metrics in the userservice, which worked as expected.